org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.releng/tp/pom.xml

```diff
@@ -43,4 +44,71 @@
 			</plugin>
 		</plugins>
 	</build>
+
+	<profiles>
+		<profile>
+			<id>tp-aggregate</id>
+			<build>
+				<plugins>
+					<plugin>
+						<artifactId>maven-clean-plugin</artifactId>
+						<version>3.0.0</version>
+						<configuration>
+							<filesets>
+								<fileset>
+									<directory>.repository</directory>
+								</fileset>
+							</filesets>
+						</configuration>
+					</plugin>
+					<plugin>
+						<groupId>org.eclipse.tycho.extras</groupId>
+						<artifactId>tycho-eclipserun-plugin</artifactId>
+						<version>${tycho-version}</version>
+						<executions>
+							<execution>
+								<goals>
+									<goal>eclipse-run</goal>
+								</goals>
+								<phase>package</phase>
+							</execution>
+						</executions>
+						<configuration>
+							<!-- IMPORTANT: Do NOT FORMAT the appArgLine, it must be in ONE line! -->
+							<appArgLine>-application org.eclipse.cbi.p2repo.cli.headless aggregate --buildModel ${targetplatform.classifier}.aggr</appArgLine>
```

Comment:
This stile is deprecated and it's better to specify each single arg in applicationsArgs, see e.g., https://github.com/LorenzoBettini/jbase/blob/master/jbase.site/pom.xml#L78